lint/linter.py

```diff
@@ -678,17 +678,38 @@ def build_cmd(self, cmd):
         return self.insert_args(cmd)
 
     def context_sensitive_executable_path(self, cmd):
-        """
-        Calculate the context-sensitive executable path, return a tuple of (have_path, path).
+        """Calculate the context-sensitive executable path.
 
-        Subclasses may override this to return a special path.
+        Subclasses may override this to return a special path. The default
+        implementation looks for a setting `executable` and if set will use
+        that.
 
         Return (True, '<path>') if you can resolve the executable given at cmd[0]
         Return (True, None) if you want to skip the linter
         Return (False, None) if you want to kick in the default implementation
             of SublimeLinter
 
+        Notable: `<path>` can be a list/tuple or str
+
```

Comment:
This is a tricky, nobody uses this yet, feature. This is a bit I tried to see the future 🎱  You can set `executable` to `/bin/flake8`, boring stuff. But you can also set it to `['pipenv', 'run', 'flake8']`. 

Unfortunately, I never had the time to test this.

Reply:
> Unfortunately, I never had the time to test this.

Me neither. I guess as soon as someone tries we'll find out if it's successful?